controllers/test_utils.go

```diff
@@ -1,18 +1,19 @@
 package controllers
 
 import (
+	"time"
+
 	"github.com/go-errors/errors"
 	. "github.com/onsi/gomega"
 	logf "sigs.k8s.io/controller-runtime/pkg/log"
-	"time"
```

Comment:
I won't block the PR on changing this but could you remove these checks from your IDE and instead use [golangci-lint](https://github.com/golangci/golangci-lint)? There is a config file in this project we will use to define the syntax rules rather than depend on the IDE.

Reply:
Okay.  After some troubleshooting I fixed the settings.